addons/website_forum/static/src/js/website_forum.js

```diff
@@ -133,18 +144,22 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
         this.$('[data-bs-toggle="tooltip"]').tooltip({delay: 0});
         this.$('[data-bs-toggle="popover"]').popover({offset: '8'});
 
-        const element = document.querySelector("div.js_select_menu_wrapper");
-        if (element) {
+        const selectMenuWrapperEl = document.querySelector("div.js_select_menu_wrapper");
+        if (selectMenuWrapperEl) {
+            const karmaError = selectMenuWrapperEl.dataset.karmaError;
+            const isReadOnly = Boolean(selectMenuWrapperEl.dataset.readonly);
             // Take default tags from the input value
             const defaultChoices = [];
-            JSON.parse(element.getAttribute("data-init-value"))?.forEach((x) => {
+            JSON.parse(selectMenuWrapperEl.getAttribute("data-init-value"))?.forEach((x) => {
                 defaultChoices.push({ name: x.name, label: x.name, value: x.id });
             });
             const defaulValue = defaultChoices.map((choice) => choice.value);
 
-            await attachComponent(this, element.parentNode, SelectMenuWrapper, {
+            await attachComponent(this, selectMenuWrapperEl.parentNode, SelectMenuWrapper, {
```

Comment:
We could keep the behavior of title if we attach the component to `selectMenuWrapperEl` instead of its parent and add the title on it.

Reply:
Yes, Let's not make it complicated. I have made the change to keep the same behavior. thanks : )